src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -145,6 +145,9 @@ class JesBackend extends Backend with LazyLogging {
     )
   }
 
+  override def engineFunctions(context: TaskExecutionContext): WdlStandardLibraryFunctions = ???
```

Comment:
Yeah we need to start making sure to account for all backends when we size these things, or split that into multiple tickets.

As a one time "oopsie" (also because they just made some serious changes to the jes backend) it's probably fine to call this a stretch goal and if incomplete make it a ticket at the top of the backlog for next sprint. Either way let's make sure Everything Else is cool and then do these.

I can help you w/ the getting started w/ jes, it's pretty quick & easy.